corpus/commands.txt

```diff
@@ -212,6 +212,10 @@ cat << "EOF"
 a=$b
 EOF
 
+cat <<"END OF FILE"
+hello
```

Comment:
Because of my comment below, I'm wondering if this test will still pass if there are multiple lines of content in the heredoc.